### ISSUE TITLE
Add private gists to especific user

### ISSUE THREAD

**Issue by anicolaspp** [MacDownApp/macdown-gistit#2]
Ask for github credentials before creating gist(s)

**Comment by uranusjr** [comment]
Yes this is what I have in mind too. I’d probably save the credential instead of prompting everything, but nothing has been decided yet.
